examples/crowd-funding/README.md

```diff
@@ -54,7 +54,7 @@ export LINERA_STORAGE1="rocksdb:$(dirname "$LINERA_WALLET1")/linera.db"
 export LINERA_WALLET2="$PWD/target/debug/wallet_2.json"
 export LINERA_STORAGE2="rocksdb:$(dirname "$LINERA_WALLET2")/linera_2.db"
 
-(cargo build && cd examples && cargo build --release)
+(cargo build && cd examples && cargo build --release && cd ..)
```

Comment:
Good catch, thank you!

But the README.md is auto-generated from the lib.rs, so the error needs to be fixed there (too).

Reply:
I see

Reply:
@afck ´cd ..´ does nothing here

Reply:
I would suggest to revert the change

Reply:
Ah, because of the brackets! (It's all in a subshell.)
I can revert it in my next PR. Sorry!

Reply:
sorry. yes subshell